doc/Language/traps.pod

```diff
@@ -248,6 +248,60 @@ The loose precedence of C<..> can lead to some errors.  It is usually best to pa
     1..3.say    # says "3" (and warns about useless "..")
     (1..3).say  # says "1..3"
 
+=head1 Subroutines
+
+=head2 Declarations
+
+Subroutines and methods are generally declared in this manner:
+
+  sub    foo(...) {} # where ... represent zero or more parameters
+  method foo(...) {} 
+
+but they may also may be declared with whitespace before the opening parenthesis:
+
+  sub    foo (...) {} # where ... represent zero or more parameters
+  method foo (...) {} 
+ 
+However, such whitespace can result in errors with certain argument
+lists when calling them using the function call form (see below).
```

Comment:
I think this is factually wrong. In declarations, whitespace between the routine name and the signature is insignificant. If you disagree, please provide an example where it makes a difference.